plugins/pdns/pdns_errors

```diff
@@ -9,11 +9,16 @@
 #%# family=auto
 #%# capabilities=autoconf
 
-command="/etc/init.d/pdns dump"
+command="/usr/bin/pdns_control list"
 
 if [ "$1" = "autoconf" ]; then
-        echo yes
-        exit 0
+        if [ -e /usr/bin/pdns_control ]; then
+                echo yes
+                exit 0
+        else
+                echo no
```

Comment:
Please return an exitcode of zero even in case of "no" (see [PluginConcise](http://munin-monitoring.org/wiki/PluginConcise#autoconf).
Additionally it would be great if you would provide a reason for "no". The documentation recommends the following pattern: `no (missing /usr/bin/pdns_control)`.